budgetportal/templates/base.html

```diff
@@ -10,18 +11,130 @@
 
 <!DOCTYPE html>
 <html>
-{% include 'utilities/html/head.html' with title=title description=description %}
-<body {{ production }} data-spy="scroll" data-offset="200">
-{% if production %}
+
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!--
+      'unsafe-inline' is added to style-src since ChartJS uses inline styling when changing the viewport.
+      If any dynamic data is piped into Jekyll it should be filtered through strip_html.
+     -->
+    <meta
+      http-equiv="Content-Security-Policy"
+      content="
+        default-src
+          'self'
+          https://vulekamali.gov.za
+          https://data.vulekamali.gov.za
+          https://discussions.vulekamali.gov.za;
+        connect-src
+          'self'
+          https://vulekamali.gov.za
+          https://data.vulekamali.gov.za
+          https://discussions.vulekamali.gov.za
+          https://appzi-collector-b.azurewebsites.net
+          https://openspending.vulekamali.gov.za
+          https://openspending.org;
+        font-src
+          https://fonts.gstatic.com/;
+        style-src
+          'self'
+          'unsafe-inline'
+          https://fonts.googleapis.com;
+        script-src
+          'self'
+          https://www.googletagmanager.com
+          https://discussions.vulekamali.gov.za
+          https://www.google-analytics.com
+          https://www.youtube.com
+          https://i.ytimg.com
+          https://app.appzi.io
+          https://d3js.org
+          {% comment %} Google Tag Manager HEAD inline {% endcomment %}
+          'sha256-v+9wGGBueHqSIuvMsUaFLaXTNw0D8igfDF+5k+9TVbE='
+        {% if debug %} {% comment %}for django-debug-toolbar{% endcomment %}
+          http://ajax.googleapis.com
+          'unsafe-eval'
+        {% endif %}
+        ;
+        frame-src *;
+        img-src
```

Comment:
```suggestion
        frame-src 'self'
                  https://app.appzi.io
                  https://www.youtube.com
                  https://discussions.vulekamali.gov.za
                  blob:;
        img-src
```

Please revert this - we don't need to relax this yet but we wanna merge this PR